### DISCUSSION TITLE
有人在中国用成功过吗？付费求指点

### DISCUSSION THREAD

**Discussion by long201058** [acheong08/ChatGPT#766] [General]
V1 Standard ChatGPT
Update 2023/02/14 9:00 PM GMT+8: It is working. Use this.
这个版本有人成功了吗

**Comment by acheong08** [comment]
我在中国。用VPN才能

**Comment by lozn00** [comment]
我测试v1成功了, 你要搞啥东西.

**Comment by Xchen-my** [comment]
codec can't encode characters in 这个是因为什么，

**Comment by yang-1027** [comment]
挂vpn可以解决，需要帮助嘛

**Comment by dotmet** [comment]
你先弄一个账号，然后把我的项目[gpt_web](https://github.com/dotmet/chatgpt_webui)部署到hugging face，就可以实现无VPN使用ChatGPT
